semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/SymbolInformationOps.scala

```diff
@@ -67,7 +67,8 @@ trait SymbolInformationOps { self: Scalacp =>
           }
           val isModuleClass = sym.entry.entryType == 10
           def isScalaObject: Boolean = isModuleClass && !sym.isJavaDefined
-          if (hasTermName || isScalaObject) k.OBJECT
+          if (sym.isPackageAccordingToClasspath) k.PACKAGE
+          else if (hasTermName || isScalaObject) k.OBJECT
```

Comment:
You can ignore the commit 0c5c84b03f2a6910376e624b828051afcfebbb72. I got greedy in 1e3a73bf326538ffa53db726b7465f815b097c50 and tried to fix an issue unrelated to this PR.